target/linux/ramips/image/mt7621.mk

```diff
@@ -337,9 +337,6 @@ define Device/hiwifi_hc5962
   DEVICE_VENDOR := HiWiFi
   DEVICE_MODEL := HC5962
   DEVICE_PACKAGES := kmod-mt7603 kmod-mt76x2 kmod-usb3 wpad-basic
-  SUPPORTED_DEVICES += hc5962
```

Comment:
Removing this will only prevent flashing without -F from stable releases, but flashing from recent snapshots after the ramips.sh removal will still work without -F.
However, I think this is acceptable in this case.